devito/ir/support/stencil.py

```diff
@@ -267,4 +267,15 @@ def __setitem__(self, key, val):
         super(Stencil, self).__setitem__(entry.dim, entry.ofs)
 
 
+def retrieve_offsets(stencils):
```

Comment:
Can/should we make this a class-level ``Stencil`` method, such as ``union`` for example?

Reply:
I don't have an opinion on that either way.

Reply:
Then yes please

Reply:
Fixed